src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
+  border: $card-border;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+  transition-duration: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    color: $color-main;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 16px 4px;
+    transition: color $transition-duration;
+  }
+
+  &__code {
+    font-weight: 400;
+    color: $color-secondary;
+    font-size: 10px;
+    line-height: 14px;
+    margin: $margin;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: $margin-left;
+    margin-right: $margin-right;
+    margin-bottom: 24px;
+    height: 16px;
+
+    &-review {
+      font-weight: 400;
+      color: $color-main;
+      font-size: 10px;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: $margin;
+  }
+
+  &__price-name {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__price-price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main;
+  }
+
+  &__buy {
+    height: 40px;
+    margin: $margin;
+    background-color: $color-bottom;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    text-decoration: none;
+    transition-duration: $transition-duration;
+
+    &:hover {
+      background-color: #fff;
+      color: $color-bottom;
```

Comment:
Ensure that the variable `$color-bottom` is defined in your SCSS files. If it's not defined, this will cause a compilation error.